### PR TITLE
Only run tests for sdk and bindings/core

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Only run tests for sdk and bindings/core

## Type of change

- Enhancement (a non-breaking change which adds functionality)

## How the change has been tested

CI will test

## Change checklist

- [ ] I have followed the contribution guidelines for this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have checked that new and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by thibault-martinez — CHANGES_REQUESTED** [review]
CI failing

File: .github/workflows/build-and-test.yml [hunk 1]
```diff
@@ -53,4 +53,4 @@ jobs:
         uses: actions-rs/cargo@v1
         with:
           command: nextest
-          args: run --all-features --no-fail-fast --cargo-profile ci --test-threads "num-cpus" --retries 2
+          args: run --all-features --no-fail-fast --cargo-profile ci --test-threads "num-cpus" --retries 2 -p iota-sdk -p iota-sdk-bindings-core
```
Comment (thibault-martinez) [inline .github/workflows/build-and-test.yml:56]: Do we want to do that also for private tangle tests ?

Comment (Thoralf-M) [reply .github/workflows/build-and-test.yml:56]: Added

Comment (DaughterOfMars) [reply .github/workflows/build-and-test.yml:56]: Change the github aliases too?

Comment (Thoralf-M) [reply .github/workflows/build-and-test.yml:56]: Done

**Review by DaughterOfMars — APPROVED** [review]

**Review by thibault-martinez — APPROVED** [review]

**Pull request merged**